docs/b2b-edition/specs/storefront/storefront/order.yaml

```diff
@@ -351,27 +357,67 @@ paths:
                 isSaveOrderComment:
                   type: string
                   description: If save order comment
-                extraFields:
-                  type: array
-                  items:
-                    type: object
-                    properties:
-                      fieldName:
-                        type: string
-                        description: Extra field name
-                      fieldValue:
-                        type: string
-                        description: Extra field value
+                extraInt1:
```

Comment:
Likewise, these extraInt, extraStr, and extraInfo fields shouldn't be added back in, as they were removed in #540